packages/block-library/src/cover/edit.native.js

```diff
@@ -370,11 +428,17 @@ const Cover = ( {
 							customColorIndicatorStyles={
 								styles.paletteColorIndicator
 							}
+							customIndicatorWrapperStyles={
+								styles.paletteCustomIndicatorWrapper
+							}
+							customVerticalSeparatorStyles={
+								styles.paletteVerticalSeparator
+							}
```

Comment:
I think this is not being used, should we remove it or is there something that we need to add?

Reply:
You are right @geriux ! This was used at some point during the development but not any more.
I updated the PR